extensions/amp-analytics/0.1/instrumentation.js

```diff
@@ -93,7 +104,61 @@ class InstrumentationService {
           }
         });
       }
+    } else if (eventType === AnalyticsEventType.CLICK) {
+      this.ensureClickListener_();
+      this.clickObservable_.add(
+          this.createSelectiveListener_(listener, opt_selector));
+    }
+  }
+
+  ensureClickListener_() {
+    if (!this.clickHandlerRegistered_) {
+      this.clickHandlerRegistered_ = true;
+      this.win_.document.documentElement.addEventListener(
+          'click', this.onClick_.bind(this));
+    }
+  }
+
+  /**
+   * @param {!Event} e
+   */
+  onClick_(e) {
+    this.clickObservable_.fire(e);
+  }
+
+  /**
+   * @param {!Function} listener
+   * @param {string=} opt_selector
+   */
+  createSelectiveListener_(listener, opt_selector) {
+    const wrappedListener = e => {
+      if (opt_selector) {
+        if (!this.matchesSelector_(e.target, opt_selector)) {
+          return;
+        }
+      }
+      listener(new AnalyticsEvent(AnalyticsEventType.CLICK));
+    };
+    return e => {
+      vsyncFor(this.win_).measure(() => {wrappedListener(e);});
+    };
+  }
+
+  /**
+   * @param {Element} el
+   * @param {string} selector
+   * @return {boolean} True if the given element matches the given selector.
+   */
+  matchesSelector_(el, selector) {
+    try {
+      const matches = this.win_.document.querySelectorAll(selector);
```

Comment:
Yep, http://caniuse.com/#search=matches

Super wide support, so we can rely on that for AMP's purpose.